main.js

```diff
@@ -63,4 +63,11 @@ for (let i = 0; i < seeProjectLinks.length; i += 1) {
 
 projects.addEventListener('click', () => {
   Toogle();
+});
+
+document.querySelector('form').addEventListener('Submit', (event) => {
+  if (event.target.email.value.match(/[A-Z]/g)) {
+    document.querySelector('span').classList.remove('message');
+    event.preventDefault();
+  }
```

Comment:
Hi @VTY1999, I will first of all congratulate you upon the work you have so far done, your project looks great! kudos 💯.  However, you still have some small changes to make.

- [ ] [REQUIRED] I have noticed that I can submit an email in capital letters. This shouldn't be the case according to the project requirement below 👇🏻 

![image](https://user-images.githubusercontent.com/4247740/147145707-ae2dede6-9960-4797-b905-cf67526840b0.png)

It would be nice if you implemented this validation so that only email addresses written in lowercase are allowed 😄.